src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/InteractiveMapsMercuryComponentObject.java

```diff
@@ -32,13 +36,15 @@
   private WebElement poiPin;
   @FindBy(css = ".leaflet-popup")
   private WebElement poiPopUp;
+  @FindBy(css = ".lightbox-content")
+  private WebElement lightbox;
 
   public InteractiveMapsMercuryComponentObject(WebDriver driver) {
     super(driver);
   }
 
   public void clickCloseButton() {
-    driver.switchTo().activeElement();
+//    driver.switchTo().activeElement();
```

Comment:
Please leave comment on why a line needs to be temporary commented out. Otherwise, remove completely.